eo-runtime/src/main/java/EOorg/EOeolang/EOtxt/EOsprintf.java

```diff
@@ -65,13 +70,63 @@ public EOsprintf(final Phi sigma) {
                     final String format = new Param(rho, "format").strong(String.class);
                     final Phi[] args = new Param(rho, "args").strong(Phi[].class);
                     final Collection<Object> items = new LinkedList<>();
-                    for (final Phi arg : args) {
-                        items.add(new Dataized(arg).take());
+                    for (int idx = 0; idx < args.length; idx += 1) {
```

Comment:
@Graur I think you can use `++idx` here

Reply:
@EugeneDar if I do so, I get checkstyle's [IllegalTokenCheck](https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenCheck.html)